src/helpers/documentation.js

```diff
@@ -220,7 +214,15 @@ export const buildDocsContents = (data, rootDir) => {
 
     addNextUpToArticles(topicArticles, 0, nextTopic);
   });
-  return contents;
+
+  /* After its nested page data are sorted and added in,
+  sort its parent by topicOrder which is metadata.data.order */
+  const sortedDocs = Object.entries(contents)
+    .sort(compareNestedEntries)
+    /* reverse it to object */
+    .reduce((acc, [k, v]) => ({ ...acc, [k]: v }), {});
```

Comment:
There's also [`Object.fromEntries`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries)

(oops I see your commit, bummer that this isn't supported in node yet)